department-of-veterans-affairs/va.gov-cms
# VACMS-19377: Fix tabbing for content editor menus.

## Description

Closes #19377

## Testing done


## Screenshots


## QA steps

When logged in to the Drupal backend, start to tab through the main menus with your keyboard (content, structure, etc).
The tabbing should work on the main menus but require you to hit enter to tab in to submenus.

As user uid with user_role

Tab through Drupal back end menus
Hit enter to enter a sub menu.
If you do not hit enter you should remain in the top level menu when tabbing

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
